src/identity/oauth.ts

```diff
@@ -113,6 +113,8 @@ export class OAuthProvider implements Provider {
     // Open browser to start authorization request
     const authBaseURL = await this.issuer.authEndpoint();
     const authURL = this.getAuthRequestURL(authBaseURL);
+    // DO NOT SUBMIT
+    // TOOD: This failure is swallowed. How do we raise this?
     await this.openURL(authURL);
```

Comment:
This rejected promise is swallowed and I am not sure where to throw it.

Reply:
I think this may be as simple as removing the `.catch` on this line:

https://github.com/github/sigstore-js/blob/main/src/sign.ts#L111

This is swallowing any errors which are thrown from within an identity provider. This code has been through a lot of refactoring and it may have made sense to swallow these errors in some past iteration, but I don't think it does anymore.